index.html

```diff
@@ -235,20 +255,31 @@
         'linkify': false
       }).use(window.markdownitTaskLists, {enabled: true});
 
-      function renderChecklist() {
-        $.ajax('/checklist.md', {
+      function renderChecklist(lang) {
+        $.ajax('/lang/' + lang + '/checklist.md', {
           method: 'GET',
           dataType: 'text'
-        }).done(function (text) {
-          $('article.entry-content').html(md.render(text));
+        }).done(function(text) {
+          $("a[data-lang='" + lang + "']").addClass('active');
+          $("article.entry-content").html(md.render(text));
           bindCheckBehavior();
-          $('li ul.task-list').parent().addClass('special-bg')
-        }).fail(function () {
-          console.error("Failed to GET checklist.md, trying again in 1 sec...")
-          setTimeout(renderChecklist, 1000)
-        });
+          $("li ul.task-list").parent().addClass("special-bg")
+        }).fail(function() {
+          console.error("Failed to GET checklist.md, is it a valid language?");
```

Comment:
Can you add `setTimeout(renderChecklist, 1000)` right below? It might be important in case the request fails for some reason like an intermittent connection.